go/libkb/saltpack.go

```diff
@@ -38,6 +38,29 @@ func (b naclBoxPublicKey) HideIdentity() bool {
 	return false
 }
 
+type hiddenNaclBoxPublicKey NaclDHKeyPublic
+
+func (b hiddenNaclBoxPublicKey) ToKID() []byte {
+	return b[:]
```

Comment:
Do I understand right that this should never be called? Maybe we should make it `panic`?